trunner/device.py

```diff
@@ -101,7 +294,7 @@ class IMXRT106xRunner(DeviceRunner):
     SDP = 'plo-ram-armv7m7-imxrt106x.sdp'
     IMAGE = 'phoenix-armv7m7-imxrt106x.disk'
 
-    def __init__(self, port, phoenixd_port='/dev/ttyUSB0'):
+    def __init__(self, port, phoenixd_port='/dev/ttyACM1'):
```

Comment:
probably we should also think about some "device configs" in the future, but it's good for now.

One tip: it might be easier to use `/dev/serial/by-path/*` or `/dev/serial/by-name/*` to always use the desired peripheral regardless of connecting order.

Reply:
Changed to `/dev/serial/by-id/*`